packages/components/pages/f-account-info/test-utils/component-objects/f-account-info.component.js

```diff
@@ -0,0 +1,134 @@
+        },
+        phoneNumber: {
+            get input () { return $(FIELDS.phoneNumber.input); },
+            get emptyError () { return $(FIELDS.phoneNumber.emptyError); },
+            get invalidError () { return $(FIELDS.phoneNumber.invalidError); }
+        },
+        addressLine1: {
+            get input () { return $(FIELDS.addressLine1.input); },
+            get emptyError () { return $(FIELDS.addressLine1.emptyError); }
+        },
+        addressLine2: {
+            get input () { return $(FIELDS.addressLine2.input); }
+        },
+        addressLine3: {
+            get input () { return $(FIELDS.addressLine3.input); }
+        },
+        city: {
+            get input () { return $(FIELDS.city.input); },
+            get emptyError () { return $(FIELDS.city.emptyError); }
+        },
+        postcode: {
+            get input () { return $(FIELDS.postcode.input); },
+            get emptyError () { return $(FIELDS.postcode.emptyError); },
+            get invalidError () { return $(FIELDS.postcode.invalidError); }
+        }
+    }
+
+    load () {
+        super.load(this.component);
+    }
+
+    click (field) {
+        this.fields[field].input.click();
+    }
+
+    waitForComponent () {
+        super.waitForComponent(this.component);
+    }
+
+    isComponentDisplayed () {
+        return this.component.isDisplayed();
+    }
+
+    /**
+    * @description
+    * Select all the text in the field and then performs a backspace to clear the field
+    *
+    * @param {String} fieldName The name of the field input it is clearing
+    */
+    clearBlurField (fieldName) {
+        // Determines the OS
+        const CONTROL = process.platform === 'darwin' ? 'Command' : '\uE009';
+        const el = this.fields[fieldName].input;
+        el.click();
+        el.keys([CONTROL, 'a']);
+        el.keys(['Backspace']);
+    }
+
+    /**
+    * @description
+    * Inputs customer details into the account-info component.
+    *
+    * @param {Object} customerInput customer input details
+    */
+    populateAccountForm (field, customerInput) {
+        this.fields[field].input.setValue(customerInput[field].input);
+    }
+
+    clickOutOfInputField () {
+        this.component.click();
+    }
+
+    isEmptyErrorMessageDisplayed (fieldName) {
+        return this.fields[fieldName].emptyError.isDisplayed();
+    }
+
+    isInvalidErrorMessageDisplayed (fieldName) {
+        return this.fields[fieldName].invalidError.isDisplayed();
+    }
+
+    emailAddressLinkCanBeClicked () {
+        return this.changeEmailAddressLink.isClickable();
+    }
+
+    saveChangesButtonCanBeClicked () {
+        return this.saveChangesButton.isClickable();
+    }
+
+    changePasswordButtonCanBeClicked () {
+        return this.changePasswordButton.isClickable();
+    }
+
+    deleteAccountLinkCanBeClicked () {
+        return this.deleteAccountLink.isClickable();
+    }
+
+    isDisabled (field) {
```

Comment:
Should this be `return !this.fields[field].input.isEnabled();`?

Reply:
ah yes thank you!